Tests/Brokerages/GDAX/GDAXBrokerageTests.cs

```diff
@@ -0,0 +1,313 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+using Moq;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using QuantConnect.Brokerages;
+using QuantConnect.Brokerages.GDAX;
+using QuantConnect.Orders;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading;
+
+namespace QuantConnect.Tests.Brokerages.GDAX
+{
+
+    [TestFixture()]
+    public class GDAXBrokerageTests
+    {
+
+        GDAXBrokerage _unit;
+        Mock<IWebSocket> _wss = new Mock<IWebSocket>();
+        Mock<IRestClient> _rest = new Mock<IRestClient>();
+        string _orderData;
+        string _orderByIdData;
+        string _openOrderData;
+        string _matchData;
+        string _accountsData;
+        string _holdingData;
+        Symbol _symbol;
+        const string _brokerId = "d0c5340b-6d6c-49d9-b567-48c4bfca13d2";
+        const string _matchBrokerId = "132fb6ae-456b-4654-b4e0-d681ac05cea1";
+
+        [SetUp()]
+        public void Setup()
+        {
+            _unit = new GDAXBrokerage("wss://localhost", _wss.Object, _rest.Object, "abc", "MTIz", "pass");
+            _orderData = File.ReadAllText("TestData\\gdax_order.txt");
```

Comment:
Could be that Travis build is failing because Linux requires `"/"` path separator instead of `"\"` ?

Reply:
Thanks, this fixed the last of the travis problems.